packages/core/src/types/data/image.ts

```diff
@@ -1,5 +1,12 @@
 import { DataLoader, UniDataset, Metadata } from './common';
 
+export interface SegmentationRLE {
+  counts: string;
+  size: number[]; 
+}
+
+export type SegmentationPolygon = number[][];
+
 export interface ImageLabel {
```

Comment:
Could we have a label named `ImageSegmentationLabel`?

Reply:
Just like coco or pascal format can handle all kinds of cv tasks. So I suggest we just have one interface for label which corresponds to coco-like format

Reply:
Ok, and I think we need detailed documentation for these fields, this might be useful for the plugin developers.

Reply:
Surely :)